### PR TITLE
Uer Guide fix

### PR DESCRIPTION
**Pull request by stephaneleong**
The commits are assocaited with the following issues:
Updated Quick Start Ui User Guide #250 #215
Update Quickstart hyperlink to release page #211
Add Constraints to addgoods command of User Guide #201 #203 #204, #248
Update address example in add command example usages #220
Update Goods unique identification rule #222
Update expectations of GUI in Quickstart #228
Format Delivery Status for clarity of example #231
Update known issues about resizing Ui elements #235
Update Tip on Tags for Adding supplier #237
Update known issues about address field bound #239
Update known issues of inconsistent capitalisation for supplier's name #240

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -263,14 +274,17 @@ This is a **key** feature of our application and deliveries are automatically ma
...
 2. **If you minimize the Help Window** and then run the `help` command (or use the `Help` menu, or the keyboard shortcut `F1`) again, the original Help Window will remain minimized, and no new Help Window will appear. The remedy is to manually restore the minimized Help Window.
+3. **UI elements can be resized to be too small**: Each UI element can be resized to a point where it may not be visible at all. If this happens accidentally, the user may be confused about the missing UI element. There should be a minimum height/width for each UI element to prevent this issue.
+4. **Unit level and number in the address field are not bounded reasonably**: For commands like `add` and `edit`, the unit level and number in the address field are not reasonably bounded. This allows users to input excessively large unit numbers without any error message.
+5. **Inconsistent Capitalisation of Supplier Name for adding Goods**: When adding goods, the supplier name follows the user's exact capitalisation of input and not the capitalisation of the supplier name in the supplier list. This may lead to confusion if the user is not aware of the exact capitalisation of the supplier name.
```
Comment (jialegitgud) [inline docs/UserGuide.md:279]: `Capitalization` should be used consistently throughout.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,20 +14,20 @@ Supply Central is a **desktop app for managing suppliers, optimized for use via
...
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103-F10-1/tp/releases).
 
 1. Copy the file to the folder you want to use as the _home folder_ for SupplyCentral.
 
 1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar SupplyCentral.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/old-Ui.png)
+   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data, goods view will only show undelivered goods upon startup.<br>
```
Comment (jialegitgud) [inline docs/UserGuide.md:22]: Use `and` instead of , for clarity.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -168,8 +168,16 @@ Format: `exit`
 
 Adds a specific goods item tied to a supplier. All fields are required for this command.
 
+
 Format: `addgoods gn/GOODS_NAME q/QUANTITY p/PRICE c/CATEGORY pd/PROCUREMENT_DATE ad/ARRIVAL_DATE n/SUPPLIER_NAME`
 
+* Goods are uniquely identified by all fields. (i.e. Goods can be repeated if any of the fields are different)
+* Quantity should be a positive number `quantity > 0`.
+* Price cannot be a negative number `price >= 0.00`.
```
Comment (jialegitgud) [inline docs/UserGuide.md:176]: use e.g. here for clarity.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -263,14 +274,17 @@ This is a **key** feature of our application and deliveries are automatically ma
...
+5. **Inconsistent Capitalisation of Supplier Name for adding Goods**: When adding goods, the supplier name follows the user's exact capitalisation of input and not the capitalisation of the supplier name in the supplier list. This may lead to confusion if the user is not aware of the exact capitalisation of the supplier name.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER a/ADDRESS [t/TAG]…​` <br> e.g. `add n/James Ho p/22224444 a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add** | `add n/NAME p/PHONE_NUMBER a/ADDRESS [t/TAG]…​` <br> e.g. `add n/James Ho p/22224444 a/125 Orchard Road #12-34 ABC Building Singapore 123456 t/friend t/colleague`
```
Comment (jialegitgud) [inline docs/UserGuide.md:287]: Change to `Add Supplier`, and same for delete/edit to explicitly show the difference. Also, tags can changed to better reflect the application theme.

**Review by jialegitgud — COMMENTED** [review]
Overall, LGTM but formatting issues have not been resolved.

**Comment by stephaneleong** [comment]
Fix formatting issues brought up. for a second review @jialegitgud

**Review by jialegitgud — APPROVED** [review]
All good

**Pull request merged**